camelConnector/src/main/java/io/vantiq/extsrc/camelconn/discover/CamelRunner.java

```diff
@@ -161,18 +168,23 @@ public Properties getCamelProperties() {
      * @throws Exception for errors during the process.  IllegalStateException if called before there is a Camel
      *                   context available.  Others may be thrown by called methods.
      */
-    private ClassLoader constructClassLoader() throws Exception {
+    private ClassLoader discoverAndBuildClassloader() throws Exception {
         if (getCamelContext() == null) {
             throw new IllegalStateException("Camel context does not exist.");
         }
         CamelDiscovery discoverer = new CamelDiscovery();
         Map<String, Set<String>> discResults = discoverer.performComponentDiscovery(routeBuilder, camelProperties);
-        
```

Comment:
some minor refactoring since we now construct a class loader with only the declared dependencies